dse-graph/build.sbt

```diff
@@ -7,19 +7,11 @@ scalaVersion := "2.12.2"
 scalacOptions += "-Ybreak-cycles"
 
 libraryDependencies ++= Seq(
-  "com.michaelpollmeier" %% "gremlin-scala" % "3.2.4.11",
-  "com.datastax.dse" % "dse-java-driver-graph" % "1.2.3",
+  "com.michaelpollmeier" %% "gremlin-scala" % "3.2.5.0",
+  "com.datastax.dse" % "dse-java-driver-graph" % "1.3.0",
   "ch.qos.logback" % "logback-classic" % "1.1.9" % Test,
   "org.scalatest" %% "scalatest" % "3.0.1" % Test
 )
 
 fork in Test := true
-
-resolvers ++= Seq(
```

Comment:
Not needed anymore as tinkerpop now depends on a jBCrypt library that is on central.